x-pack/plugins/security_solution/public/flyout/document_details/right/footer.tsx

```diff
@@ -182,8 +178,13 @@ export const PanelFooter: FC<PanelFooterProps> = ({ isPreview }) => {
 
   return (
     <>
-      <StyledEuiFlyoutFooter data-test-subj={FLYOUT_FOOTER_TEST_ID}>
-        <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlyoutFooter data-test-subj={FLYOUT_FOOTER_TEST_ID} className="side-panel-flyout-footer">
```

Comment:
I'm curious about why adding back the `side-panel-flyout-footer` class here? I had removed it in [this PR](https://github.com/elastic/kibana/pull/189289) back in August. I just removed it and testing and the color is still here

Reply:
the color missing came from the fact that `styled('div')` was used instead of `styled('EuiFlyoutFooter')`

Reply:
i thought so initially, but the padding was needed inside the footer to create the gap around the action button
<img width="690" alt="image" src="https://github.com/user-attachments/assets/4145b8e4-d1f1-4689-aabc-f26f4cb714f1">

Reply:
> I'm curious about why adding back the side-panel-flyout-footer class here? I had removed it in https://github.com/elastic/kibana/pull/189289 back in August. I just removed it and testing and the color is still here

Added it back because it was showing a slight gap between the footer and the scrollbar, I thought it fixed it. However I cannot reproduce that gap, removed in 1e9dc053815324a30594302d9886f64efbe9c3b2